sbin/spark-daemon.sh

```diff
@@ -78,6 +78,12 @@ spark_rotate_log ()
     if [ -n "$2" ]; then
 	num=$2
```

Comment:
@srowen 
There provide a param,num, to set the number of logfile,but it wasn't used in line 179.Since it doesn't work,can I remove the param "num" and use it just as a variable？